ros:
  cam0_topic: "/rs/ir0/image"
  cam1_topic: "/rs/ir1/image"
  imu0_topic: "/rs/imu0/data"

settings:
  data_path: "/tmp/calib_data"
  info_gain_threshold: 0.5

calib_target:
  target_type: 'aprilgrid'  # Target type
  tag_rows: 6               # Number of rows
  tag_cols: 6               # Number of cols
  tag_size: 0.0375          # Size of apriltag, edge to edge [m]
  tag_spacing: 0.3          # Ratio of space between tags to tagSize
                            # Example: tagSize=2m, spacing=0.5m --> tagSpacing=0.25[-]

cam0:
  resolution: [640, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"

cam1:
  resolution: [640, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"
